vispy/util/fonts/tests/test_font.py

```diff
@@ -3,39 +3,42 @@
 # Distributed under the (new) BSD License. See LICENSE.txt for more info.
 
 import numpy as np
-from nose.tools import assert_true, assert_equal
 import warnings
 
 from vispy.testing import assert_in, run_tests_if_main
 from vispy.util.fonts import list_fonts, _load_glyph, _vispy_fonts
+import pytest
 
-warnings.simplefilter('always')
+known_bad_fonts = set([
+    'Noto Color Emoji',  # https://github.com/vispy/vispy/issues/1771
+])
+
+# try both a vispy and system font   <--- what does this mean???
+sys_fonts = set(list_fonts()) - set(_vispy_fonts)
 
 
 def test_font_list():
     """Test font listing"""
     f = list_fonts()
-    assert_true(len(f) > 0)
+    assert len(f) > 0
     for font in _vispy_fonts:
         assert_in(font, f)
 
 
-def test_font_glyph():
+@pytest.mark.parametrize('face', ['OpenSans'] + sorted(sys_fonts))
+def test_font_glyph(face):
     """Test loading glyphs"""
-    # try both a vispy and system font
-    sys_fonts = set(list_fonts()) - set(_vispy_fonts)
-    # assert_true(len(sys_fonts) > 0) not always the case!
-    for face in ['OpenSans'] + sorted(list(sys_fonts)):
-        print(face)  # useful for debugging
```

Comment:
The print statement is no longer useful since the font name will appear sure to parameterization

Reply:
Awesome!